doc-content/enterprise-only/openshift/jboss-postconfigure-proc.adoc

```diff
@@ -0,0 +1,92 @@
+[id="jboss-postconfigure-proc_{context}"]
+= (Optional) Providing Elytron user configuration or other post-configuration settings
```

Comment:
I wouldn't put Optional in this way. We haven't discussed how to signal procedures that are option but I don't think we should put it in the title of the module. Perhaps add a note.